interface/batchcom/batchcom.php

```diff
@@ -24,280 +29,217 @@
-                    require_once ('batchPhoneList.php');
-                break;
+                case $process_choices[0]: // CSV File
+                    generate_csv($res);
+                    exit();
+                case $process_choices[1]: // Email
+                    require_once('batchEmail.php');
+                    exit();
+                case $process_choices[2]: // Phone list
+                    require_once('batchPhoneList.php');
+                    exit();
             endswitch;
         }
-        // end results
 
-        exit ();
     }
 }
 
-//START OUT OUR PAGE....
 ?>
 <html>
 <head>
 <title><?php echo xlt('BatchCom'); ?></title>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" href="batchcom.css" type="text/css">
-<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.min.css">
-
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-3-1-1/index.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker-2-5-4/build/jquery.datetimepicker.full.min.js"></script>
-
-<script LANGUAGE="JavaScript">
-$(document).ready(function() {
-    $('.datepicker').datetimepicker({
-        <?php $datetimepicker_timepicker = false; ?>
-        <?php $datetimepicker_showseconds = false; ?>
-        <?php $datetimepicker_formatInput = false; ?>
-        <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
-        <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
-    });
-});
-</script>
-
+<?php Header::setupHeader(['datetime-picker']); ?>
+<style>
+.datepicker {
+    width: 100%;
+}
+</style>
 </head>
+
 <body class="body_top">
 <!-- larry's sms/email notification -->
-<span class="title"><?php include_once("batch_navigation.php");?></span>
+<?php include_once("batch_navigation.php");?>
 <!--- end of larry's insert -->
-<span class="title"><?php xl('Batch Communication Tool','e')?></span>
-<br><br>
-
-<!-- for the popup date selector -->
-<div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>
-<FORM name="select_form" METHOD=POST ACTION="" onsubmit='return top.restoreSession()'>
-<div class="text">
-<div class="main_box">
-    <table class="table" ><tr><td >
-        <?php
-        if ($form_err) {
-            echo (xl('The following errors occurred')."<br>$form_err<br><br>");
+<main class="container">
+    <header class="row">
+        <div class="col-md-6 col-md-offset-3 text-center">
+            <h1><?php xl('Batch Communication Tool','e')?></h1>
+        </div>    
+    </header>
+    <?php if ($form_err) { echo "<div class=\"alert alert-danger\">".xl("The following errors occurred").": $form_err</div>"; } ?>
```

Comment:
Would a ternary statement be clear here, or is several lines the preferable option?